tests/mlio-test/test_image_reader.cxx

```diff
@@ -0,0 +1,439 @@
+#include <gtest/gtest.h>
```

Comment:
I think some of these tests would benefit from having comments added to clarify what is being tested. It would also be helpful in the future if someone uses these tests as reference for using the C++ API.

Reply:
I have named each test in a way that makes its purpose clear, and have tried to follow the `Arrange Act Assert` pattern. What type of comments are you referring to here? Do you mean comments within the tests? Just making sure that my understanding is correct :)

Reply:
Yup, just comments within the test. If most of the tests are similar, maybe you only need to do so for a couple of them.